src/jsd/epd_actuator.cc

```diff
@@ -280,6 +280,12 @@ void fastcat::EpdActuator::ElmoRead()
          sizeof(jsd_epd_state_t));
 }
 
+void fastcat::EpdActuator::ElmoFault()
+{
+  // TODO review with dloret
```

Comment:
After looking at David's PRs, I think this is not a problem for the Platinum as of right now.

It seemed that the problem was with the values passed to the `trap` functions, which are only used during calibration.